services/index/src/api.js

```diff
@@ -69,6 +70,11 @@ builder.declare({
   route: '/namespaces/:namespace?',
   query: paginateResults.query,
   name: 'listNamespaces',
+  scopes: {
+    if: 'namespaceGiven',
+    then: 'index:list-namespaces:<namespace>',
+    else: 'index:list-namespaces',
```

Comment:
This would mean that the anonymous scopes need `index:list-namespaces` and `index:list-namespaces:*`, or `index:list-namespaces*`.  Is that the best option?

Reply:
Per our conversation on Slack, I will go ahead and change this to only require one scope `index:list-namespaces:<namespace>'. A user/service would need `index:list-namespaces:*` to view all namespaces.